src/App.tsx

```diff
@@ -1,9 +1,55 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import classNames from 'classnames';
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { wait } from './utils/fetchClient';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [search, setSearch] = useState<string>('');
+  const [sortBy, setSortBy] = useState<Status>(Status.All);
+
+  const titleInput = useRef<HTMLInputElement>(null);
+
+  const isAllCompleted = todos.every(todo => todo.completed);
+
+  const activeTodosCount = todos.filter(todo => !todo.completed).length;
+
+  let visibleTodos = todos.filter(todo =>
+    todo.title.toLowerCase().includes(search.toLowerCase()),
+  );
```

Comment:
input shouldn't work as a search
in the next stage, we will use it as input to add new todo

Reply:
I know that you know it) but it looks a bit confusing